src/inject.js

```diff
@@ -41,13 +47,17 @@ const injectStyleTag = (cssContents /* : string */) => {
             styleTag.type = 'text/css';
             styleTag.setAttribute("data-aphrodite", "");
             head.appendChild(styleTag);
+        } else if (styleTag.styleSheet) {
+            // $FlowFixMe: legacy Internet Explorer compatibility
+            styleString = styleTag.innerText;
         }
     }
 
 
     if (styleTag.styleSheet) {
+        styleString += cssContents;
         // $FlowFixMe: legacy Internet Explorer compatibility
-        styleTag.styleSheet.cssText += cssContents;
+        styleTag.styleSheet.cssText = styleString;
```

Comment:
I agree that we should avoid having to keep track of `styleString` if possible, but that solution won't quite work because `innerText` doesn't get updated when `cssText` is altered.